iron-icon.html

```diff
@@ -89,6 +100,7 @@
 
       width: var(--iron-icon-width, 24px);
       height: var(--iron-icon-height, 24px);
+      color: var(--iron-icon-color, inherit);
```

Comment:
I've merged https://github.com/PolymerElements/iron-icon/pull/39 which adds custom properties for `fillColor` and `strokeColor`. Is this still needed?